### ISSUE TITLE
Not working in action modal

### ISSUE THREAD

**Issue by martin-ro** [savannabits/filament-flatpickr#6]
When using the field in an action modal I get

```
Alpine Expression Error: mode is not defined

Expression: "mode ==='dark'"

<template x-if="mode ==='dark'">
```

**Comment by martin-ro** [comment]
I should add that this is with a table action outside of admin panel

**Comment by coolsam726** [comment]
Hi Martin,
Kindly share your component's code to help me reproduce the issue. Thanks.

**Comment by martin-ro** [comment]
Hi,

it's just a livewire component implementing `Tables\Contracts\HasTable` and in `getTableActions`

```php
Tables\Actions\Action::make('book')
                ->label(__('Book Lesson'))
                ->mountUsing(fn (Forms\ComponentContainer $form, Teacher $record) => $form->fill([
                    'teacher_id' => $record->user->userable_id,
                ]))
                ->form([
                    Group::make()->schema([
                        Flatpickr::make('date),
                    ])->columns(2),
                ])
```

**Comment by martin-ro** [comment]
If you want to reproduce, just create a Livewire component and setup a table like [here](https://filamentphp.com/docs/2.x/tables/getting-started#preparing-your-livewire-component)

and in the tables `getTableActions` create a modal action with the Flatpickr field. I hope you know what I mean? I can't really post the whole component :)

**Comment by martin-ro** [comment]
Ok, digging around a bit, all the assets are registered `FlatpickrServiceProvider` which means they are only loaded within Filament but not in a standalone form or table component.

**Comment by coolsam726** [comment]
> Ok, digging around a bit, all the assets are registered `FlatpickrServiceProvider` which means they are only loaded within Filament but not in a standalone form or table component.

This is actually the case. Might you be able to do a PR to support standalone usage?

**Comment by R00118189** [comment]
Any update?

**Comment by coolsam726** [comment]
I will be re-writing most parts of the component tomorrow, hopefully it
will fix this issue.

**Comment by R00118189** [comment]
.

**Comment by dkstudio86** [comment]
> @coolsam726 I wonder if you had a chance to update this package to work with standalone forms

+1

**Comment by Haggai-Mac** [comment]
> @coolsam726 I wonder if you had a chance to update this package to work with standalone forms

+1

**Comment by coolsam726** [comment]
Fixed in this PR #27
